packages/onboarding/src/flow-progress/use-flow-progress.ts

```diff
@@ -1,4 +1,4 @@
-import { LINK_IN_BIO_FLOW, LINK_IN_BIO_TLD_FLOW } from '../utils/flows';
+import { LINK_IN_BIO_FLOW, LINK_IN_BIO_TLD_FLOW, ECOMMERCE_FLOW } from '../utils/flows';
```

Comment:
Extreme nit: I'd prefer this to be in alphabetical order.
```suggestion
import { ECOMMERCE_FLOW, LINK_IN_BIO_FLOW, LINK_IN_BIO_TLD_FLOW } from '../utils/flows';
```
And this is _definitely_ not blocking!